dask_expr/tests/test_collection.py

```diff
@@ -154,6 +154,14 @@ def test_dropna(pdf):
     assert_eq(df.y.dropna(), pdf.y.dropna())
 
 
+def test_fillna():
+    pdf = pd.DataFrame({"x": [1, 2, None, None, 5, 6]})
+    df = from_pandas(pdf, npartitions=2)
+    actual = df.fillna(value=100).compute()
```

Comment:
It's better not to call `.compute()` explicitly in Dask tests unless we have to. Instead we let `assert_eq` do this.  That way `assert_eq` can do things like verify that the columns and dtypes that Dask thought it would get are what actually happened.  It can also run various sanity checks on the task graph to make sure that names are sensible and so on.

If you call compute ahead of time then `assert_eq` doesn't have the Dask graph to actually ask questions of

```suggestion
    actual = df.fillna(value=100)
```